device-types/Nokia/7210-SAS-T-12F10T-4XFP.yaml

```diff
@@ -0,0 +1,71 @@
+---
+manufacturer: Nokia
+model: 7210 SAS-T 12F10T 4XFP
+slug: nokia-7210-sas-t-12f10t-4xfp
+part_number: 3HE08116AAAB01
+u_height: 2
+is_full_depth: false
+subdevice_role: parent
```

Comment:
`subdevice_role: parent` is only to be used if this device is either a parent (i.e. a chassis device that houses other devices), or or a child (i.e. this device slots into a chassis device). Generally parent devices have `device-bays` defined for their children devices.

**If this is not actually a parent device, please remove this definition**